src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios.md

```diff
@@ -56,20 +56,21 @@ export const options = {
 
 For each k6 scenario, the VU workload is scheduled by an _executor_.
 For example, executors configure:
+- How long the test runs
 - Whether VU traffic stays constant or changes
 - Whether to model traffic by iteration number or by VU arrival rate.
 
 Your scenario object must define the `executor` property with one of the predefined executors names.
 Along with the generic scenario options, each executor object has additional options specific to its workload.
-For the list of the executors, refer to the [Executor guide](/using-k6/scenarios/executors/).
+For the list of the executors, refer to [Executors](/using-k6/scenarios/executors/).
```

Comment:
@MattDodsonEnglish I think we should list here all the executors. If not, readers might move to Concepts without getting the "general" idea of the different executor options.

I suggest adding something similar than the [Executors table](https://k6.io/docs/using-k6/scenarios/executors/). For example:

- [Shared iterations](https://k6.io/docs/using-k6/scenarios/executors/shared-iterations): a fixed amount of iterations are "shared" between a number of VUs. 
- [Per VU iterations](https://k6.io/docs/using-k6/scenarios/executors/per-vu-iterations): each VU executes an exact number of iterations. 
- ...

Reply:
@ppcano I deleted that list in https://github.com/grafana/k6-docs/pull/936. I didn't like duplicating content in this way.

I'll make a new PR to put it back in, maybe just as a summary:

You can configure executors in to distribute workload according to:
- Iterations. Either shared by the VUs, or distributed across them.
- VUs. Either a constant number or a ramping number.
- Iterations per second. Either constant or ramping.

Reply:
>I'll make a new PR to put it back in

#985